app.py

```diff
@@ -20,27 +16,6 @@
 bot = telebot.TeleBot(os.environ.get(config.env_key, None))
 
 
-@bot.message_handler(commands=['punch'])
-@with_remove(bot)
-def send_punch(message):
-    phrase = random.choice(config.phrases).capitalize()
-    bot.send_message(message.chat.id, phrase)
-
-
-@bot.message_handler(commands=['sayhialbert'])
-@with_remove(bot)
-def send_hi(message):
-    sticker = random.choice(config.stickers)
-    bot.send_sticker(message.chat.id, sticker)
-
-
-@bot.message_handler(commands=['fact'])
-@with_remove(bot)
-def send_fact(message):
-    fact = get_fact()
-    bot.send_message(message.chat.id, fact)
-
-
 @bot.message_handler(commands=['pubg'])
 @with_remove(bot)
 def send_pubg_request(message):
```

Comment:
You're right, but `send_pubg_request` now has a more complex structure than other phrases, at least it has `markup_message` and etc.. Although we can do this on the new structure too, but it would take longer than rewriting the other phrases. Maybe in next relise)